### PR TITLE
[2.9] updating import cluster test to be less flaky

### PR DESCRIPTION
**Pull request by slickwarren**
## Issue: 

 import cluster test has been flaky. 

This PR adds a wait to getting the kubeconfig. This is sometimes a problem that I noticed during testing

it also restricts the test to only run the corral that will run the test. This is to help refine exactly what runs as part of the test.

### CODE REVIEW

**Review by markusewalker — DISMISSED** [review]
LGTM

**Review by susesgartner — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -120,7 +120,7 @@ require (
 	github.com/prometheus/client_golang v1.16.0
 	github.com/prometheus/client_model v0.4.0
 	github.com/prometheus/common v0.44.0
-	github.com/rancher/aks-operator v1.3.0-rc11
```
Comment (slickwarren) [inline go.mod:123]: this shouldn't happen. I think its because shepherd go.mod isn't updated to 11 yet.

**Review by markusewalker — APPROVED** [review]

**Review by Josh-Diamond — APPROVED** [review]

**Review by susesgartner — APPROVED** [review]
LGTM

**Pull request merged**